packages/block-editor/src/components/use-setting/index.js

```diff
@@ -132,7 +132,7 @@ export default function useSetting( path ) {
 				blockName
 			);
 
-			if ( undefined !== result ) {
+			if ( undefined !== result || null !== result ) {
```

Comment:
Can we just do this instead?

```suggestion
			if ( ! result ) {
```

This is one drawback to JavaScript.

Reply:
Sure :) I thought it had an purpose, that `result` was checked that explicit there. But I would suggest to solve it on both `if` statements like this.

Reply:
I changed it in the other `if` statements as well. I hope it fits better now?

Reply:
![image](https://github.com/WordPress/gutenberg/assets/32584944/3935b665-f63c-4770-9728-f129742181a0)

One more thing: I noticed, that the last commits on `trunk` failed. So my changes may be innocent.